clj/resources/lang/en.edn

```diff
@@ -288,6 +288,8 @@
    :slipstream.ui.views.user.alert.no-ssh-keys-configured.others-profile.title  "No SSH keys configured in this profile"
    :slipstream.ui.views.user.alert.no-ssh-keys-configured.others-profile.msg    "This user will not be able to access the deployed machines via SSH until at least one SSH public key has been configured in this profile."
 
+   :slipstream.ui.views.user.footnote-about-adding-connectors! "<strong>To add or remove connectors <a href='%1$s?edit=true#close-sections'>edit</a> this page</strong> (i.e. your user profile). You will find additionnal available connectors highlighted in green. To disable a connector remove its credentials while you are in edit mode."
+
```

Comment:
Is the `!` at the end of `:slipstream.ui.views.user.footnote-about-adding-connectors!` on purpose?